### DISCUSSION TITLE
New field for editing translations

### DISCUSSION THREAD

**Discussion by Zrki85** [snipe/snipe-it#13358] [Ideas]
@snipe Is there a possibility to insert an option in the settings for each language so that we can manually translate each word. That we can translate as we want ourselves, and not depend on someone else's translation and wait for something to be corrected?
Tnx in advanced

**Comment by snipe** [comment]
The level of complexity required there would be untenable, unfortunately. You can always edit the language files yourself and/or contribute to the CrowdIn translation project if you don’t feel a translation is correct. Ping me if you want to become a proofreader.

**Reply by Zrki85** [reply]
Tnx for the fast response. Can you tell me how i can edit the language by myself?
